src/main/java/com/redhat/devtools/lsp4ij/features/documentSymbol/LSPDocumentSymbolStructureViewModel.java

```diff
@@ -136,19 +186,20 @@ public LSPDocumentSymbolViewElement(DocumentSymbolData documentSymbolData) {
         }
 
         private @NotNull Collection<StructureViewTreeElement> collectElements(@Nullable DocumentSymbolData documentSymbolData) {
-            var children = documentSymbolData.getChildren();
-            if (children.length == 0) {
+            var children = documentSymbolData != null ? documentSymbolData.getChildren() : null;
```

Comment:
And again, a combination of being defensive against `null` values and applying recommended code changes.